### ISSUE TITLE
[a11y] CTScan WebLinkButton text field missing

### ISSUE THREAD

**Issue by grantfitzsimmons** [specify/specify7#1131]
**Describe the bug**
The WebLinkButton for CTScan is missing its text field on a11y. 

These WebLinkButtons cannot be activated with keyboard navigation.

**To Reproduce**
1. Open the CO form.
2. See error

**Expected behavior**
There should be a text field for the CT Scan WebLinkButton.

**Screenshots**
## a11y
![image](https://user-images.githubusercontent.com/37256050/146947945-d0691ad4-f49b-4c31-9362-7509193634de.png)

## edge
![image](https://user-images.githubusercontent.com/37256050/146948026-c793d7b1-82a5-4dd1-aa26-2cf9d33eb7da.png)

**Desktop (please complete the following information):**
 - OS: Windows 10 & macOS Monterey
 - Browser: Edge
